CHANGELOG.adoc

```diff
@@ -12,6 +12,9 @@
 - [core] Records are now used as the implementations of `IInput`
 - https://github.com/eclipse-sirius/sirius-components/issues/1574[#1574] [diagram] In `diagram.graphqls`, `SingleClickOnDiagramElementTool` member's `targetDescriptions` is now of new type `DiagramElementDescription` instead of `NodeDescription`.  
 
+=== Bug fixes
+- https://github.com/eclipse-sirius/sirius-components/issues/1607[#1607] [tree] Cannot reopen a recently close representation
+
```

Comment:
This may not be useful since we are fixing a but introduced during this iteration, not a bug from a previous iteration